code/modules/paperwork/paper.dm

```diff
@@ -108,220 +147,87 @@
 	user.visible_message("<span class='suicide'>[user] scratches a grid on [user.p_their()] wrist with the paper! It looks like [user.p_theyre()] trying to commit sudoku...</span>")
 	return (BRUTELOSS)
 
+/// ONLY USED FOR APRIL FOOLS
+
```

Comment:
This gap may cause issues with autodoc? @SpaceManiac

Reply:
It won't break the tool, but it will look better without spacing

Reply:
Ill fix the spacing  not sure how the space got there